src/main/resources/auth0.yaml

```diff
@@ -0,0 +1,10 @@
+clientId: "KTzgyxs6KBcJHB872eSMe2cpTHzhxS99"
+clientSecret: "tjAwhBit0deVUor3lcd7YRlHRc_4FJwv_fcLp_ozO2-D2Lh38066IUoGPL4zx1vP"
```

Comment:
Oooohhhh I hope these aren't real :'(

Reply:
We discussed on slack, but no they're not real.